hw/ip/flash_ctrl/data/flash_ctrl.sv.tpl

```diff
@@ -40,7 +42,8 @@ module flash_ctrl import flash_ctrl_pkg::*; #(
   output       lc_flash_rsp_t lc_o,
   input        pwrmgr_pkg::pwr_flash_req_t pwrmgr_i,
   output       pwrmgr_pkg::pwr_flash_rsp_t pwrmgr_o,
-  input        edn_entropy_t edn_i,
+  output       edn_pkg::edn_req_t edn_o,
+  input        edn_pkg::edn_rsp_t edn_i,
```

Comment:
in this module is EDN synchronous (same clock as IP's), or it will be changed later?

Reply:
yeah this one is synchronous at the moment. Although i may decide to add `prim_reqack` later just to make it more self contained

Reply:
should we support both sync and async mode for EDN in DV?

Reply:
technically for block level testing we should probably support both...
what are you guys doing for other blocks though? Are you doing both or just
supporting the usage on top_earlgrey for now?

Reply:
right now I'm doing it for block level. It's good that each IP spec can clarify what mode it supports for EDN
Haven't thought about top-level too much. Since top-level is SW test, I think we may verify it through CSR to control the IP clocks. @sriyerg may have a better picture for this.

Reply:
i'll push a separate PR to have `prim_reqack`.  With the latest discussion, i think this is the most wholesome way to manage it anyways.

Reply:
sounds good, thanks